README.md

```diff
@@ -1,8 +1,26 @@
-RaiBlocks
-======
-RaiBlocks is designed to be a low latency, high throughput cryptocurrency https://raiblocks.net
+<img src="logo.png" width="200"/>
+
+# What is RaiBlocks?
+RaiBlocks is designed to be a feeless, instant, high throughput cryptocurrency.
 
 We've applied the philosophy of "Do one thing and do it well" giving you performance and scalability unmatched by any other platform.
 
-Build instructions:
+___
+
+### Key Features:
+* RaiBlocks utilizes a novel block-lattice architecture, unlike conventional blockchains used in many other cryptocurrencies.
+* The network requires minimal resources, no high-power mining hardware, and can process high transaction throughput.
+* Offers feeless, instantaneous transactions, as well as unlimited scalability, making RaiBlocks ideal for peer-to-peer transactions.
+* To date, the RaiBlocks network has processed over four million transactions with an unpruned ledger size of only 1.7GB.
```

Comment:
Please change "To date" to something like "As of December 2017"

Reply:
Fixed!